README.md

```diff
@@ -1197,7 +1197,7 @@ end.save
 
 TODO: Implement this!
 
-Now that you have a bunch of objects with spatial coordinates, it would be nice to find out which objects are closest to a point. This can be done by using a GeoPoint data type with query on the field using $nearSphere. Getting a list of ten places that are closest to a user may look something like:
+Now that you have a bunch of objects with spatial coordinates, it would be nice to find out which objects are closest to a point. This can be done by using a GeoPoint data type with query on the field using $nearSphere. Getting a list of ten places that are closest to an user may look something like:
```

Comment:
[Indeed](http://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user)